utopia-remix/app/routes/v1.project.$id.tsx

```diff
@@ -20,8 +23,47 @@ export async function loader(args: LoaderFunctionArgs) {
 export async function action(args: ActionFunctionArgs) {
   return handle(args, {
     PUT: {
+      handler: createProject,
+      validator: ALLOW,
+    },
+    POST: {
       handler: proxy,
       validator: ALLOW,
     },
   })
 }
+
+async function createProject(req: Request, params: Params<string>) {
```

Comment:
Note: this currently is not transactional, but I think we can address it later.

Reply:
yes, I think ideally we would want to proxy code (actual server code) to create the project and set the access level in the same transaction